packages/components/src/components/shell-panel/shell-panel.tsx

```diff
@@ -58,6 +58,18 @@ export class ShellPanel extends LitElement {
    */
   messages = useT9n<typeof T9nStrings>();
 
+  private sizeOverrideController = useSizeOverride({
```

Comment:
For consistency, let's omit the `Controller` suffix (see https://github.com/Esri/calcite-design-system/pull/11943#issuecomment-3104563277).